test/test_prototype_transforms_consistency.py

```diff
@@ -663,6 +663,15 @@ def test_call_consistency(config, args_kwargs):
 def test_get_params_alias(config):
     assert config.prototype_cls.get_params is config.legacy_cls.get_params
 
+    if not config.args_kwargs:
+        return
```

Comment:
just filter in the parametrization above?

Reply:
That would also skip the `assert` above. So if we want to filter in the parametrization, we need two tests.

Reply:
I missed that, thanks